Lib/test/test_email/data/msg_47.txt

```diff
@@ -0,0 +1,10 @@
+MIME-Version: 1.0
+Content-Type: multipart/alternative;
+ boundary="----5F0D91A9C8C24D91AF71E1D7F2F7877E"
+
+------5F0D91A9C8C24D91AF71E1D7F2F7877E
+Content-type: text/plain; charset=utf-8
+
+Test if UTF-8 messages with no Content-Transfer-Encoding set can be as_string'd:
+Föö bär
+------5F0D91A9C8C24D91AF71E1D7F2F7877E--
```

Comment:
I would prefer to have the message text encoded in the test file rather than introduce a new file in the data directory.  I think it makes the tests easier to read and understand.  You'll find a number of other tests where I've done that elsewhere in the file.